pkg/lib/context-menu.scss

```diff
@@ -0,0 +1,17 @@
+.contextMenu {
+  position: fixed;
+  /* xterm accessibility tree has z-index 100 and we need to be in front of it
+     * to be able to handle mouse events.
+     */
+  z-index: 101;
```

Comment:
I also had a translate to offset the position to make the menu appear under the mouse cursor's active area instead of being a bit off. That's missing here.

Reply:
Not sure what the translate should be. Can't see it being removed.